services/backend/src/routes/users.js

```diff
@@ -83,4 +84,19 @@ router.post('/language', async (req, res) => {
   }
 })
 
+router.get('/from-sisu-by-eppn/:newUserEppn', auth.roles(['admin']), async (req, res) => {
+  let { username: requesterEppn } = req.user
+  const { newUserEppn } = req.params
+  // in order to test this feature in the dev environment we need to set an eppn that demo sisu will recognize
+  if (isDev && requesterEppn === 'mluukkai') requesterEppn = newUserEppn
+  const person = await userService.getUserFromSisuByEppn(requesterEppn, newUserEppn)
+  res.json(person)
+})
+
+router.post('/add', async (req, res) => {
```

Comment:
Should this route also contain `auth.roles(['admin'])`?

Reply:
Yup, fixed that!